### PR TITLE
oops

### PR DESCRIPTION
**Pull request by rntz**
Fix mistaken use of square brackets in `talon_draft_windows/draft_window.talon`.

### CODE REVIEW

**Comment by richardmcmillen** [comment]
The intent was to have cursorless style phrasing in the draft editor.

I agree that this isn't, totally correct, would the cursorless way of doing
this be?

> change air say some words

To replace the word at hint 'air' with the literal phrase 'some words'.

If you think this is different enough or confusing you can revert it.

**Review by EdJoJob — APPROVED** [review]

**Comment by pokey** [comment]
No worries! To be clear, I am not advocating for one thing or another here, just saying how it works for Cursorless. I have no objection to doing things differently in the draft editor

I do agree with @rntz that "change ... with ..." is a bit awkward, but that doesn't necessarily mean you need to mimic Cursorless exactly. Eg switching to "change ... to ..." could be reasonable.

The main reason we don't support that grammar in Cursorless, fwiw, is that we prefer to allow people to use their own / knausj insertion commands, as they are fairly sophisticated / might be custom

But again, I'm happy to defer to @splondike / other maintainers on this one as I'm not an active user of the draft editor

**Pull request merged**
